src/ctapipe/irf/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""Top level module for the irf functionality"""
+from .benchmarks import (
+    AngularResolution2dMaker,
+    AngularResolutionMakerBase,
```

Comment:
You should probably avoid exposing the base classes here, as those are internal to the package